src/backend/marsha/core/serializers.py

```diff
@@ -527,3 +567,19 @@ def get_url(self, obj):
             )
 
         return url
+
+    def update(self, instance, validated_data):
```

Comment:
There is a `validate_{field}` method for this.
https://www.django-rest-framework.org/api-guide/serializers/#validation

Reply:
I don't want to validate it, I never throw an exception, I don't understand why I should use a validation method instead of this one.

Reply:
The `validate_{field}` methods return a value (after eventual modification like in your case...). This is their job and they do it at the right time. Your code will also be cleaner at the field value is passed to you directly and just needs to be returned...